services/web/client/compile.json

```diff
@@ -2,12 +2,7 @@
   "targets": [
     {
       "type": "source",
-      "outputPath": "source-output",
-      "bundle": {
-        "include": [
-          "qx.*"
-        ]
-      }
+      "outputPath": "source-output"
```

Comment:
@oetiker all good here?

Reply:
@ignapas could you please check this please???